rfc/graphql-schema/additionalTypeDefs/byPage/quickOrder.ts

```diff
@@ -0,0 +1,159 @@
+export default /* GraphQL */ `
+
+type PurchasedProductOptionType {
+    label: String!
```

Comment:
couldn't we use the same type as `Orders` endpoint? Essentially that's what these are - products that are from your orders.